lib/player.js

```diff
@@ -7121,6 +7121,14 @@ shaka.Player = class extends shaka.util.FakeEventTarget {
           'statuses. If we don\'t have all the keys, you\'ll need to set ' +
           'restrictions so we don\'t select those tracks.');
     }
+    const isEmptyKeyId = keyIds.length === 1 && keyIds[0] === '';
```

Comment:
This seems like a non-standard version of isGlobalStatus above (keyIds[0] == '00').

Could you merge them instead?  Then any application reading key statuses will see a recognizable key that is not specific to this platform.

Maybe something like:

```js
let keyIds = Object.keys(keyStatusMap);

// ...

if (keyIds.length == 1 && keyIds[0] == '') {
  keyStatusMap = {'00': keyStatusMap['']};
  keyIds = ['00'];
}

const isGlobalStatus = ...
```